packages/@aws-cdk/aws-redshift-alpha/lib/private/handler-props.ts

```diff
@@ -25,6 +25,7 @@ export interface TableHandlerProps {
 }
 
 export interface TablePrivilege {
+  readonly tableId: string;
```

Comment:
Pretty much, it's not exported, there it will not be a breaking change. However we should therefore let the user know to update their CLI version I guess?

Reply:
I've removed this